content-scripts/inject/run-userscript.js

```diff
@@ -6,22 +6,28 @@ export default async function runAddonUserscripts({ addonId, scripts, enabledLat
   for (const scriptInfo of scripts) {
     const { url: scriptPath, runAtComplete } = scriptInfo;
     const scriptUrl = `${new URL(import.meta.url).origin}/addons/${addonId}/${scriptPath}`;
-    console.log(
-      `%cDebug addons/${addonId}/${scriptPath}: ${scriptUrl}, runAtComplete: ${runAtComplete}`,
-      "color:red; font-weight: bold; font-size: 1.2em;"
-    );
     const loadUserscript = async () => {
       await scratchAddons.l10n.loadByAddonId(addonId);
       const module = await import(scriptUrl);
-      const log = _realConsole.log.bind(console, `%c[${addonId}]`, "color:darkorange; font-weight: bold;");
-      const warn = _realConsole.warn.bind(console, `%c[${addonId}]`, "color:darkorange font-weight: bold;");
       const msg = (key, placeholders) =>
         scratchAddons.l10n.get(key.startsWith("/") ? key.slice(1) : `${addonId}/${key}`, placeholders);
       msg.locale = scratchAddons.l10n.locale;
+      scratchAddons.console.log(
+        { _consoleAddonId: `${addonId} [page]` },
+        `Running ${scriptUrl}, runAtComplete: ${runAtComplete}, enabledLate: ${enabledLate}`
+      );
+      const localConsole = {
+        log: (log) => scratchAddons.console.log({ _consoleAddonId: addonId }, log),
+        debug: (log) => scratchAddons.console.debug({ _consoleAddonId: addonId }, log),
+        error: (log) => scratchAddons.console.error({ _consoleAddonId: addonId }, log),
+        info: (log) => scratchAddons.console.info({ _consoleAddonId: addonId }, log),
+        warn: (log) => scratchAddons.console.warn({ _consoleAddonId: addonId }, log),
+        table: (log) => scratchAddons.console.table({ _consoleAddonId: addonId }, log),
+      };
       module.default({
         addon: addonObj,
         global: globalObj,
-        console: { ..._realConsole, log, warn },
+        console: localConsole,
```

Comment:
```suggestion
        console: {...console, ...localConsole},
```

Reply:
Why would you add the regular console variable instead of localConsole? localConsole is the version with all of the styling while console is unstyled. This would also throw an error (I think) because console and localConsole have keys of the same name.

Reply:
It will start with `console` then overide them with `localConsole`. So if you try to access a key not in `localConsole` it will fall back to `console`.

Reply:
Oh ok, my bad.